packages/dataviews/src/components/dataviews/stories/fixtures.tsx

```diff
@@ -652,6 +652,9 @@ export const fields: Field< SpaceObject >[] = [
 		type: 'text',
 		enableHiding: false,
 		enableGlobalSearch: true,
+		filterBy: {
```

Comment:
One question I'm wondering, if I don't define this what happens? do we have default operators.